packages/twenty-front/src/modules/ui/layout/hooks/__tests__/useShowAuthModal.test.tsx

```diff
@@ -67,6 +67,17 @@ const testCases = [
   { loc: AppPath.Verify, isLogged: true, subscriptionStatus: SubscriptionStatus.Active, onboardingStatus: OnboardingStatus.InviteTeam, res: false },
   { loc: AppPath.Verify, isLogged: true, subscriptionStatus: SubscriptionStatus.Active, onboardingStatus: OnboardingStatus.Completed, res: false },
 
+  { loc: AppPath.VerifyEmail, isLogged: true, subscriptionStatus: undefined, onboardingStatus: OnboardingStatus.PlanRequired, res: true },
+  { loc: AppPath.VerifyEmail, isLogged: true, subscriptionStatus: SubscriptionStatus.Canceled, onboardingStatus: OnboardingStatus.Completed, res: false },
```

Comment:
Are you sure about these? I felt like the behavior of VerifyEmail was independent of login or anything related to the currently logged in user

Reply:
Good catch, you're absolutely right! I've fixed this in a recent commit. 

Please resolve this once you see it. Thanks!